crates/runtime/src/model/mod.rs

```diff
@@ -26,6 +26,7 @@ mod tool_use;
 
 pub use chat::{try_to_chat_model, LLMModelStore};
 pub use embed::{try_to_embedding, EmbeddingModelStore};
+pub use tool_use::ToolUsingChat;
```

Comment:
Made public so it could be used for chat model creation without the need to re-instantiate the underlying model and restart runtime configuration.